packages/sms-46elks/plugin.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable no-console */
+import { Plugin, setupProvider } from '@zemble/core'
+import yoga from '@zemble/graphql'
+
+import type { IStandardSendSmsService } from '@zemble/core'
+
+interface Sms46ElksConfig extends Zemble.GlobalConfig {
+  readonly ELKS_USERNAME?: string
+  readonly ELKS_PASSWORD?: string
+  readonly disable?: boolean
+  readonly options: {
```

Comment:
Could be nice to add link to the Elk46 docs here https://46elks.se/docs/send-sms#:~:text=(response.text)-,Optional%20request%20parameters,message,-Enable%20to%20avoid